.github/workflows/transports-release.yml

```
@@ -231,6 +231,25 @@ jobs:
+          if [ "${{ job.status }}" == "success" ]; then
+            TITLE="🚀 **Transport Release Complete**"
+            STATUS="✅ Success"
+            TRANSPORT_VERSION_LINE="**Transport Version**: \`${{ needs.build-transports.outputs.transport_version }}\`"
+            CORE_VERSION_LINE="**Core Version**: \`${{ needs.build-transports.outputs.core_version }}\`"
+            MESSAGE="$TITLE\n**Status**: $STATUS\n$TRANSPORT_VERSION_LINE\n$CORE_VERSION_LINE\n**Branch**: \`main\`\n**Commit**: \`${{ github.sha }}\`\n**Author**: ${{ github.actor }}\n**[View Workflow Run](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})**"
+          else
+            TITLE="🚀 **Transport Release Failed**"
+            STATUS="❌ Failed"
+            MESSAGE="$TITLE\n**Status**: $STATUS\n**Branch**: \`main\`\n**Commit**: \`${{ github.sha }}\`\n**Author**: ${{ github.actor }}\n**[View Workflow Run](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})**"
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)